cpp/open3d/t/geometry/PointCloud.h

```diff
@@ -364,6 +364,12 @@ class PointCloud : public Geometry, public DrawableGeometry {
     std::tuple<PointCloud, core::Tensor> RemoveRadiusOutliers(
             size_t nb_points, double search_radius) const;
 
+    /// \brief Assigns uniform color to the point cloud.
+    ///
+    /// \param color  RGB color for the point cloud. {3,} shaped Tensor.
+    /// Floating color values are clipped between 0.0 and 1.0.
+    PointCloud PaintUniformColor(const core::Tensor &color) const;
+
```

Comment:
What is the consideration to return the copy of the painted point cloud? Do we want to keep the original unpainted pointcloud?